### ISSUE TITLE
[question] LTX video latent normalization cause bad reconstruct result, why use normalization?

### ISSUE THREAD

**Issue by eisneim** [a-r-r-o-w/finetrainers#168]
### Feature request  / 功能建议

this is just a question 
```python
        latents = vae.encode(image_or_video).latent_dist.sample(generator=generator)
        latents = latents.to(dtype=dtype)
        _, _, num_frames, height, width = latents.shape
        # this cause bad reconstruct results
        # latents = _normalize_latents(latents, vae.latents_mean, vae.latents_std)
        latents = _pack_latents(latents, patch_size, patch_size_t)
```
## when using normalization

https://github.com/user-attachments/assets/9556f5a0-c8a1-4ae3-a2ad-8cd50972a44a

## without normalization

https://github.com/user-attachments/assets/f047ca30-9795-4bc1-839b-4a571e563b3e

```
from ltx_video_lora import *
device = "cuda"
dtype = torch.bfloat16
# ------------------- 

vae = load_latent_models()["vae"].to(device, dtype=dtype)

def _unpack_latents(
        latents: torch.Tensor, num_frames: int, height: int, width: int, patch_size: int = 1, patch_size_t: int = 1
    ) -> torch.Tensor:
    batch_size = latents.size(0)
    latents = latents.reshape(batch_size, num_frames, height, width, -1, patch_size_t, patch_size, patch_size)
    latents = latents.permute(0, 4, 1, 5, 2, 6, 3, 7).flatten(6, 7).flatten(4, 5).flatten(2, 3)
    return latents

file = "/home/eisneim/www/ml/video_gen/ltx_training/data/ltxv_disney_49x768x512/VID_20230606_101912.pt"
data = torch.load(file)
ll = data["latents"][0]
ll.shape

lt = _unpack_latents(ll.unsqueeze(dim=0).to(device, dtype=dtype), 7, 512//32,  768//32)
print(ll.shape)
lt.shape

timestep = torch.tensor([0.0], device=device, dtype=dtype)

with torch.no_grad():
    video =  vae.decode(lt, timestep, return_dict=False)[0]
pcc = VideoProcessor(vae_scale_factor=32)
vv = pcc.postprocess_video(video)[0]
export_to_video(vv, "data/test_rec_0.91_no_norm.mp4")
```
if no normalization applied will the lora training get better results?? @a-r-r-o-w

### Motivation / 动机

better training 

### Your contribution / 您的贡献

none, just some testing

**Comment by eisneim** [comment]
Ops my bad, after de-normalize the latent, the reconstructed video looks ok now
```
latents = latents * latents_std / scaling_factor + latents_mean
```
